.travis.yml

```diff
@@ -1,48 +1,25 @@
-language: go
-go:
-  - 1.5.1
-env:
-  - GO15VENDOREXPERIMENT=1
+language: generic
 branches:
   only:
     - master
+env:
+  DEV_REGISTRY=quay.io
 cache:
   directories:
-    - $HOME/.cache/pip
-    - $GOPATH/src/github.com/deis/workflow/vendor
-    - $HOME/venv
+    - vendor
 services:
   - docker
-  - postgresql
 sudo: required
-addons:
-  postgresql: "9.3"
-before_install:
-  - go get github.com/golang/lint/golint
-  - wget "http://ftp.debian.org/debian/pool/main/s/shellcheck/shellcheck_0.3.7-5_amd64.deb"
-  - sudo dpkg -i shellcheck_0.3.7-5_amd64.deb
-  - sudo pip install virtualenv
-  - virtualenv -p python3.5 $HOME/venv
-  - source $HOME/venv/bin/activate
-  - createdb -U postgres deis
 install:
-  - pip install -r rootfs/requirements.txt
-  - pip install -r rootfs/dev_requirements.txt
-  - GLIDE_HOME=/home/travis/.glide make -C client bootstrap
+  - make bootstrap
 script:
-  - make test
-  - make -C client/ bootstrap build test dist prep-bintray-json
+  - make build test dist prep-bintray-json
 deploy:
-  - provider: script
+  - provider: bintray
     # ensure client/doc builds aren't removed
     # see https://docs.travis-ci.com/user/deployment/#Uploading-Files
     skip_cleanup: true
-    script: _scripts/deploy.sh
```

Comment:
@mboersma good point - done in https://github.com/deis/workflow-cli/commit/cb8ee509338301f0b94a955eb0c4402393c1422c